src/søknad/steg/4-barnasbosted/AnnenForelderKnapper.tsx

```diff
@@ -33,42 +35,46 @@ const AnnenForelderKnapper: React.FC<Props> = ({
     detAndreBarnet: IBarn
   ) => {
     settBarnHarSammeForelder(true);
-    const denAndreForelderen = cloneDeep(detAndreBarnet.forelder);
+    const oppdatertForelder = cloneDeep(detAndreBarnet.forelder);
     oppdaterAnnenForelder(detAndreBarnet.id);
 
     settForelder({
-      ...forelder,
-      id: denAndreForelderen?.id,
-      navn: denAndreForelderen?.navn,
-      fødselsdato: denAndreForelderen?.fødselsdato,
-      ident: denAndreForelderen?.ident,
-      borINorge: denAndreForelderen?.borINorge,
-      hvordanPraktiseresSamværet:
-        denAndreForelderen?.hvordanPraktiseresSamværet,
-      borAnnenForelderISammeHus: denAndreForelderen?.borAnnenForelderISammeHus,
+      ...barn.forelder,
+      id: oppdatertForelder?.id,
+      navn: oppdatertForelder?.navn,
+      fødselsdato: oppdatertForelder?.fødselsdato,
+      ident: oppdatertForelder?.ident,
+      borINorge: oppdatertForelder?.borINorge,
+      borAnnenForelderISammeHus: oppdatertForelder?.borAnnenForelderISammeHus,
       borAnnenForelderISammeHusBeskrivelse:
-        denAndreForelderen?.borAnnenForelderISammeHusBeskrivelse,
-      boddSammenFør: denAndreForelderen?.boddSammenFør,
-      flyttetFra: denAndreForelderen?.flyttetFra,
-      hvorMyeSammen: denAndreForelderen?.hvorMyeSammen,
-      beskrivSamværUtenBarn: denAndreForelderen?.beskrivSamværUtenBarn,
-      land: denAndreForelderen?.land,
-      fraFolkeregister: denAndreForelderen?.fraFolkeregister,
+        oppdatertForelder?.borAnnenForelderISammeHusBeskrivelse,
+      boddSammenFør: oppdatertForelder?.boddSammenFør,
+      flyttetFra: oppdatertForelder?.flyttetFra,
+      hvorMyeSammen: oppdatertForelder?.hvorMyeSammen,
+      beskrivSamværUtenBarn: oppdatertForelder?.beskrivSamværUtenBarn,
+      land: oppdatertForelder?.land,
+      fraFolkeregister: oppdatertForelder?.fraFolkeregister,
     });
   };
 
   const leggTilAnnenForelder = () => {
     settBarnHarSammeForelder(false);
-    oppdaterAnnenForelder('annen-forelder');
```

Comment:
Har du testet at denne endringen ikke brekker visningen av valget for barnets andre forelder?
https://github.com/navikt/familie-ef-soknad/pull/1338

Reply:
Ja, har bare flyttet hvor `annen-forelder` legges til. Skjer på linje 74 i samme fil nå.